package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "",
   "main": "src/index.js",
```

Comment:
Like this: https://github.com/eggheadio/egghead-io-styleguide/blob/master/package.json#L5